trigger/process.go

```diff
@@ -289,3 +296,57 @@ projectLoop:
 
 	return versions, catcher.Resolve()
 }
+
+// TriggerDownstreamProjectsForPush triggers downstream projects for a push event from a repo that does not
+// have repotracker enabled.
+func TriggerDownstreamProjectsForPush(ctx context.Context, projectId string, event *github.PushEvent, processor projectProcessor) ([]model.Version, error) {
+	downstreamProjects, err := model.FindDownstreamProjects(projectId)
+	if err != nil {
+		return nil, errors.Wrapf(err, "finding downstream projects of project '%s'", projectId)
+	}
+
+	catcher := grip.NewBasicCatcher()
+	versions := []model.Version{}
+projectLoop:
```

Comment:
Do we need this? I think we can just break at line 347

Reply:
Nope, will remove - I was following the patterns from `triggerDownstreamProjectsForBuild` and `triggerDownstreamProjectsForTask`. I changed them both to remove the projectLoop as well